test/functional/test_framework/test_framework.py

```diff
@@ -90,6 +90,9 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
 
     This class also contains various public and private helper methods."""
 
+    chain: str
```

Comment:
We can't use these annotations while we are still supporting Python 3.5.

Reply:
Yes, that's unfortunately true. However, Python 3.5 EOL is 09/2020. Is that a sufficient reason to bump Python version, or what is the policy for upgrade here?